docs/scopes.md

```diff
@@ -21,6 +21,7 @@ Die nachfolgende Tabelle stellt eine Liste, der aktuell verfügbaren Scopes dar.
 | ` baufinanzierung:echtgeschaeft ` |  BaufiSmart-Echtgeschäft bearbeiten  |    |
 | ` baufinanzierung:vorgang:lesen ` |  BaufiSmart-Vorgänge lesen  |    |
 | ` baufinanzierung:vorgang:schreiben ` |  BaufiSmart-Vorgänge anlegen  |    |
+| ` baufinanzierung:vorgang:loeschen ` |  BaufiSmart-Vorgänge löschen  |    |
```

Comment:
Hi @ElisaBaum,
die Änderung in der `scopes.md` brauchst du hier nicht. 
https://github.com/europace/authorization-api/blob/master/.github/workflows/scopes-doc.yaml kümmert sich mittels GitHub Action darum